recipes/neo4j-python-driver/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name='neo4j-python-driver' %}
+{% set version='1.1.0rc1' %}
```

Comment:
Because py2neo (for reasons I don't understand myself) specifically requires this release 1.1.0rc1.